src/django_components_js/README.md

```diff
@@ -26,7 +26,7 @@ Components.registerComponentData(
 Components.callComponent(
   "table",  // Component name
   12345,    // Component ID - An HTML element with corresponding
-            //                attribute (`data-comp-id-12345`) MUST
+            //                attribute (`data-djc-id-12345`) MUST
```

Comment:
I changed the attribute that will hold component's IDs from `data-comp-id-` to `data-djc-id-`.